sys-process/gotop/gotop-3.0.0.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit golang-vcs-snapshot
+
+EGO_PN="github.com/cjbassi/${PN}"
+
+DESCRIPTION="A terminal based graphical activity monitor inspired by gtop and vtop"
+HOMEPAGE="https://github.com/cjbassi/gotop"
+
+SRC_URI="https://github.com/cjbassi/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE="AGPL-3"
```

Comment:
Please list LICENSEs for all bundled libraries. You can use `dev-util/ninka` to automatically check what particular LICENSE file is.